src/config/statsd.ts

```diff
@@ -21,7 +21,7 @@ const statsd = new StatsD({
 	globalTags,
 	errorHandler: (err) => {
 		if (process.env.NODE_ENV !== EnvironmentEnum.development) {
-			logger.warn(err, "error writing metrics");
+			logger.warn({...err}, "Error writing metrics");
```

Comment:
I wonder why are we doing it?

Reply:
because when an `Error` type object is sent to bunyan, it seems to only display 3 properties of the error (stack, type, msg) and ignore the rest.  I'm trying to make sure that it displays everything.

Reply:
Yes, but if we do `{...err}` the stack trace will be lost.

We need to do something like this to get all fields:
`	logger.error(err, `Webhook Error: ${JSON.stringify(err)}`)`

(Maybe there is more elegant way to do it, but the err need to be the first parameter to get stack trace)

Reply:
Fixed this by modifying the global serializers for bunyan.